src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml

```diff
@@ -22,6 +22,17 @@
             <tag name="cache.pool" />
         </service>
 
+        <service id="cache.adapter.system" class="Symfony\Component\Cache\Adapter\ChainAdapter" abstract="true">
+            <factory class="Symfony\Bundle\FrameworkBundle\FrameworkBundle" method="createSystemCache" />
```

Comment:
I would move this factory to the component instead as the code in the method is not tied to the full-stack framework. I'd like to be able to reuse this strategy into Silex for instance.